docs/data/faostat.md

```diff
@@ -232,29 +232,38 @@ Optionally, set `INSPECT_ANOMALIES=True`, to visualize if anomalies that were de
 ```bash
```

Comment:
I think before here you need to add instructions related to country harmonization, because if not there are errors. The `excluded_countries` file is not generated:
```
(etl-py3.11) parriagadap@Pablos-MacBook-Pro etl % etl garden/faostat/2023-04-04
Detecting which steps need rebuilding...
OK (3s)                                                               
Running 9 steps:
1. data://garden/faostat/2023-04-04/faostat_metadata...
2023-04-04 10:04:55,535 - [bugsnag] WARNING - No API key configured, couldn't notify
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/parriagadap/etl/.venv/lib/python3.11/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/parriagadap/etl/.venv/lib/python3.11/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/Users/parriagadap/etl/.venv/lib/python3.11/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/parriagadap/etl/.venv/lib/python3.11/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/parriagadap/etl/etl/run_python_step.py", line 34, in main
    _import_and_run(path, dest_dir)
  File "/Users/parriagadap/etl/etl/run_python_step.py", line 52, in _import_and_run
    module.run(dest_dir)
  File "/Users/parriagadap/etl/etl/steps/data/garden/faostat/2023-04-04/faostat_metadata.py", line 1005, in run
    excluded_countries = io.load_json(excluded_countries_file)
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/parriagadap/etl/.venv/lib/python3.11/site-packages/owid/datautils/decorators.py", line 64, in wrapper_download
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/parriagadap/etl/.venv/lib/python3.11/site-packages/owid/datautils/io/json.py", line 50, in load_json
    with open(json_file, "r") as _json_file:
         ^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/parriagadap/etl/etl/steps/data/garden/faostat/2023-04-04/faostat.excluded_countries.json'

COMMAND: poetry run run_python_step data://garden/faostat/2023-04-04/faostat_metadata /Users/parriagadap/etl/data/garden/faostat/2023-04-04/faostat_metadata
```

And if I copy the file it asks for a new country that appeared (French Guiana), so adding something like 
```
harmonize data/meadow/faostat/YYYY-MM-DD/faostat*.feather country etl/steps/data/garden/faostat/YYYY-MM-DD/faostat.countries.json
```
would help.

Reply:
After adding `"French Guiana": "French Guiana"` I got a duplicated index error, so I can't move forward. I see you need to make adjustments to the code each time the FAO updates this.
```
(etl-py3.11) parriagadap@Pablos-MacBook-Pro etl % etl garden/faostat/2023-04-04
Detecting which steps need rebuilding...
OK (3s)                                                               
Running 9 steps:
1. data://garden/faostat/2023-04-04/faostat_metadata...
 75%|██████████████████████████████████████████████████████████████████████████████                          | 21/28 [00:12<00:02,  2.81it/s]2023-04-04 10:09.42 [warning  ] 0 item codes in faostat_rp missing in metadata. 38 item codes in data mapping to different items in metadata.
100%|████████████████████████████████████████████████████████████████████████████████████████████████████████| 28/28 [00:28<00:00,  1.03s/it]
2023-04-04 10:09.58 [warning  ] 6 domains have changed titles, consider updating custom_datasets.csv.
2023-04-04 10:09:58,748 - [bugsnag] WARNING - No API key configured, couldn't notify
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/parriagadap/etl/.venv/lib/python3.11/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/parriagadap/etl/.venv/lib/python3.11/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/Users/parriagadap/etl/.venv/lib/python3.11/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/parriagadap/etl/.venv/lib/python3.11/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/parriagadap/etl/etl/run_python_step.py", line 34, in main
    _import_and_run(path, dest_dir)
  File "/Users/parriagadap/etl/etl/run_python_step.py", line 52, in _import_and_run
    module.run(dest_dir)
  File "/Users/parriagadap/etl/etl/steps/data/garden/faostat/2023-04-04/faostat_metadata.py", line 1036, in run
    countries_table = create_table(df=countries_df, short_name="countries", index_cols=["area_code"])
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/parriagadap/etl/etl/steps/data/garden/faostat/2023-04-04/faostat_metadata.py", line 700, in create_table
    table = table.set_index(index_cols, verify_integrity=True)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/parriagadap/etl/vendor/owid-catalog-py/owid/catalog/tables.py", line 289, in set_index
    t = super().set_index(
        ^^^^^^^^^^^^^^^^^^
  File "/Users/parriagadap/etl/.venv/lib/python3.11/site-packages/pandas/util/_decorators.py", line 331, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/parriagadap/etl/.venv/lib/python3.11/site-packages/pandas/core/frame.py", line 6068, in set_index
    raise ValueError(f"Index has duplicate keys: {duplicates}")
ValueError: Index has duplicate keys: CategoricalIndex([69], categories=[1, 2, 3, 4, ..., 99029, 99030, 99031, 99032], ordered=False, dtype='category', name='area_code')

COMMAND: poetry run run_python_step data://garden/faostat/2023-04-04/faostat_metadata /Users/parriagadap/etl/data/garden/faostat/2023-04-04/faostat_metadata
```